examples/05_joining_tables_with_FuzzyJoin.py

```diff
@@ -358,3 +361,98 @@ def print_worst_matches(joined_table, n=5):
+# Instantiating the transformer
+# ............................
+# To do this, we will first gather the auxilliary tables into a
+# list of (tables, keys):
+
+aux_wb_tables = [
+    (gdppc, "Country Name"),
+    (life_exp, "Country Name"),
+    (legal_rights, "Country Name"),
+]
+
+y = df["Happiness score"]
+#######################################################################
+# We have our auxilliary tables in a list!
+# Let us now create an instance of the transformer with the necessary information:
+from dirty_cat import FeatureAugmenter
+
+fa = FeatureAugmenter(tables=aux_wb_tables, main_key="Country")
+
+#################################################################
+# Fitting and transforming into the final table
+# ............................................
+# To get our final joined table we will fit and transform the main table (df)
+# with our create instance of the |fa|:
+df_final = fa.fit_transform(df)
+
+df_final.head(10)
+
+##########################################################################
+# And that's it! As previously, we now have a big table
+# ready for machine learning.
+# Let's create our machine learning pipeline:
+from sklearn.pipeline import make_pipeline
+from sklearn.compose import make_column_transformer
+
+# We include only the columns that will be pertinent for our regression:
+encoder = make_column_transformer(
+    (
+        "passthrough",
+        [
+            "GDP per capita (current US$)",
+            "Life expectancy at birth, total (years)",
+            "Strength of legal rights index (0=weak to 12=strong)",
+        ],
+    ),
+    remainder="drop",
+)
+
+pipeline = make_pipeline(fa, encoder, HistGradientBoostingRegressor())
+
+##########################################################################
+# And the best part is that we are now able to evaluate the paramaters of the |fj|.
+# For instance, the ``match_score`` was manually picked and can now be
+# introduced into a grid search:
+
+from sklearn.model_selection import GridSearchCV
+
+# We will test four possible values of match_score:
+params = {"featureaugmenter__match_score": [0.2, 0.3, 0.4, 0.5]}
+
+grid = GridSearchCV(pipeline, param_grid=params)
+grid.fit(df, y)
+
+print(grid.best_params_)
+##########################################################################
+# The grid searching gave us the best value of 0.5 for the parameter
+# ``match_score``. Let's use this value in our regression:
+#
+
+print(f"Mean R2 score with pipeline is {grid.score(df, y):.2f}")
+
+##########################################################################
+#
+# .. topic:: Note:
+#
+#    Here, ``grid.score()`` takes directly the best model
+#    (with ``match_score=0.5``) that was found in previous iterations.
+#    Thus, it is equivalent to fixing the ``match_score`` to 0.5 and
+#    refitting the pipeline on the data.
+#
+#
+# Great, by evaluating the correct ``match_score`` we improved our
+# results significantly!
```

Comment:
```suggestion
# Tuning the ``match_score`` parameter of the |fa| improved prediction 
# performance markedly!
```

Reply:
Given how important it seems to tune this parameter, it might be useful to mention it in the doc string of the FeatureAugmenter, suggestion to use hyper-parameter tuning tools on it.